src/test/scala/it/pagopa/interop/partyprocess/service/impl/SignatureSpec.scala

```diff
@@ -0,0 +1,56 @@
+package it.pagopa.interop.partyprocess.service.impl
+
+import it.pagopa.interop.partyprocess.model.UserRegistryUser
+import it.pagopa.interop.partyprocess.server.impl.Dependencies
+import it.pagopa.interop.partyprocess.service.SignatureValidationService
+import org.scalatest.Ignore
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+import java.nio.file.{Files, Path, Paths}
+import java.util.UUID
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+@Ignore
+class SignatureSpec extends AnyWordSpecLike with Matchers with Dependencies {
```

Comment:
does this test require any trick to be executed? 
in case may you add a comment with instructions?